adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/rest/v1/FormatEndpoint.java

```diff
@@ -0,0 +1,25 @@
+package org.jvalue.ods.adapterservice.adapter.rest.v1;
+
+import org.jvalue.ods.adapterservice.adapter.AdapterFactory;
+import org.jvalue.ods.adapterservice.adapter.interpreter.Interpreter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+@RestController
+public class FormatEndpoint {
```

Comment:
Just moved to other package to separate stateless execution from config management